src/frontend/src/app/components/home/home.component.html

```diff
@@ -1 +1,99 @@
-<p>home works!</p>
+<main>
+
+
+  <h2>Ласкаво просимо у фонд гуртожитків Львівського національного університету!</h2>
+  <div id="first-image">
+    <img src="https://lh3.googleusercontent.com/p/AF1QipMTHsHeJXh-6iYNLKopGseGll17QVk6MDq27W0j=s0"
+         alt="Гуртожиток"
+         style="margin-top: -150px; margin-bottom: -50px" />
+  </div>
+  <p>
+    Цей сайт розроблений студентами для студентів, а отже, саме тут ви знайдете всю правду про наші «общаги».
+    Не треба шукати квартиру та віддавати господарям всю стипендію, коли рідний універ має стільки гуртожитків.
+    Не зволікайте та <a href="/request_admission">заселяйтесь</a>!
+  </p>
+  <ul>
+    <li>2 гуртожитки поряд із легендарною <a href="https://uk.wikipedia.org/wiki/Медова_печера">Медовою печерою</a></li>
+    <li>4 гуртожитки на межі мальовничого <a href="https://uk.wikipedia.org/wiki/Парк_«Погулянка»">лісопарку «Погулянка»</a></li>
+    <li>1 гуртожиток у тихій місцевості <a href="https://uk.wikipedia.org/wiki/Голоско">Голоско</a></li>
+    <li>1 елітний гуртожиток у самісінькому центрі міста</li>
+  </ul>
+
+  <div align="center" style="margin-bottom: 2em;">
+    <a href="/dorms">
+      <button>
+        Докладніше про гуртожитки
+      </button>
+    </a>
+  </div>
+
+  <div id="second-image">
+    <img src="https://lh3.googleusercontent.com/p/AF1QipN5DbCjz46XrqaBlG1dqdWLx9dd4ZdU7BZ1lc8_=s0"
+         alt="Гуртожиток"/>
+  </div>
+
+  <ul style="margin-left: 3em;">
+    <li>Демократичні ціни</li>
+    <li>Американська мрія: long commute to work</li>
+    <li>Веселі сусіди</li>
+    <li>Ручні тарганчики</li>
+    <li>Спортзали</li>
+    <li>Загартування холодною водою</li>
+    <li>Школа виживання</li>
+  </ul>
+
+  <div align="center">
+    <a href="/request_admission">
+      <button>
+        Порахувати ціни
+      </button>
+    </a>
+  </div>
+
+  <h2>Галерея</h2>
+  <br clear="all" />
+  <div class="my-grid">
+    <div class="image-wrapper">
+    <img
+         src="https://lh3.googleusercontent.com/p/AF1QipPAOVUHlnmGPnOEKheGybSlAcY0JiIHnv5An5EY=s0"
+         alt="Табличка"/>
+    </div>
+    <div class="image-wrapper">
+    <img
+         src="https://lh3.googleusercontent.com/p/AF1QipP3CCMQtJPj2G91s2BfUxcWq6sJASlEzf226I8w=s0"
+         alt="Гуртожиток"/>
+    </div>
+    <div class="image-wrapper">
+    <img
+         src="https://lh3.googleusercontent.com/p/AF1QipOtioedSshIvB7f5F_yP_fHK-w--2oeLqVyUq3Q=s0"
+         alt="Кухня"/>
+    </div>
+    <div class="image-wrapper">
+    <img
+         src="https://students.lnu.edu.ua/campus/wp-content/uploads/2015/11/12443080_666154136855770_1102748597_n-1.jpg"
+         alt="Спортзал"/>
+    </div>
+  </div>
+</main>
+
+<footer>
```

Comment:
You should move this footer to the bottom of app.component.html, so that it appears on all pages

Reply:
I thought this footer is suitable only for main page, because this page has no Disqus as a logical end. If you think it should be everywhere, let it be.